src/main/resources/view/PersonListCard.fxml

```diff
@@ -55,8 +55,8 @@
         <Label fx:id="moneyPaid" styleClass="cell_small_label" text="\$moneyPaid" />
       </HBox>
       <HBox>
-        <Label styleClass="cell_small_label" text="Additional Notes: " minWidth="105"/>
-        <Label fx:id="additionalNotes" styleClass="cell_small_label" text="\$additionalNotes" wrapText="true" />
+        <Label styleClass="cell_small_label" text="Additional Notes: " />
+        <Label fx:id="additionalNotes" styleClass="cell_small_label" text="\$additionalNotes" />
```

Comment:
I think you may need to pull from upstream and merge into your branch, because it looks like you are reverting the changes back to the old one haha. ie: ln 58 and 59 should include `minWidth=105` and `wrapText=true`